export_img.py

```diff
@@ -37,33 +34,21 @@ def find_inkscape_path():
 
 inkscape = find_inkscape_path()
 
-def create_png_directory():
-    """
-    Check if png directory exist, if not create new one
-    """
-    dir = "../png"
-    if os.path.isdir(dir):
-        pass
-    else:
-        os.mkdir(dir)
-
 def export_svg_png():
     """
     Export img.svg to img.png with Inkscape
     """
-    for img in base.glob("*.svg"):
+    for img in Path("./img").glob("**/*.svg"):
```

Comment:
Zde by bylo fajn aspon trochu vedet, co to dela.

Reply:
takže do dokumentačního řetězce vysvětlit tento řádek? nebo jen komentář vedle toho?